tensorwaves/physics/helicityformalism/kinematics.py

```diff
@@ -1,16 +1,262 @@
+r"""Kinematic based calculations for the helicity formalism.
```

Comment:
We may have to discuss the file structure further. For instance, if other formalisms are implemented, something like this may be more convenient:
```python
from tensorwaves.physics.formalism.helicity import Kinematics
```
(In that case, though `class HelicityKinematics` would have to be defined as `class Kinematics(interfaces.Kinematics)`.)

Something similar applies to for instance `PhaseSpaceGenerator`, I mean, that you can just replace their implementation by modifying the imports without for instances having to replace all instances of `HelicityFormalism` in your script.

But I'm not sure how conventional such a structure where we rely more on namespaces would be. The motivation is initially more the file structure, but also that you normally don't mix formalisms or generators (although not even sure about that).

Reply:
Lets move this to a separate discussion. Can you open an issue for that and link it here.

Reply:
Hm perhaps better not to pursue this indeed:
> Flat is better than nested.
https://inventwithpython.com/blog/2018/08/17/the-zen-of-python-explained/

Reply:
But I do think it's better to use `tensorwaves.physics.formalism.helicity` instead of `tensorwaves.physics.helicityformalism`. Or maybe `formalisms`. Then `physics` has at least `dynamics` and `formalisms`.

Reply:
I think the dynamics is also strongly coupled to the formalism

Reply:
Okay leave it like this